examples/python-dataplugin-examples/hello-world/README.md

```diff
@@ -0,0 +1,3 @@
+# Hello World Example

```

Comment:
I can no longer see "Hello World" in the example

Reply:
right, let's call the group "Hello World"